tests/WebAppCallsWebApiCallsGraph/TodoListService/Startup.cs

```diff
@@ -33,7 +33,7 @@ public void ConfigureServices(IServiceCollection services)
             // Adds Microsoft Identity platform (AAD v2.0) support to protect this Api
             services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                     .AddMicrosoftIdentityWebApi(Configuration, "AzureAd")
-                        .CallsWebApi()
+                        .CallsWebApis()
```

Comment:
Renaming to express that this is a hub for downstream APIs.
Note that CallsWebApis() really provides the ITokenAcquisitionService .... but I don't think we want to write that?